Marlin/ultralcd_impl_HD44780.h

```diff
@@ -811,12 +811,23 @@ static void lcd_implementation_status_screen() {
     #if LCD_WIDTH >= 20 && ENABLED(SDSUPPORT)
 
       lcd.setCursor(7, 2);
-      lcd_printPGM(PSTR("SD"));
-      if (IS_SD_PRINTING)
-        lcd.print(itostr3(card.percentDone()));
-      else
-        lcd_printPGM(PSTR("---"));
-      lcd.write('%');
+      if (blink) {
+        lcd_printPGM(PSTR("Fs"));
+        lcd.print(itostr3(fanSpeeds[0]));
+        lcd.write(' ');
+      }
+      else {
+        if (IS_SD_PRINTING) {
+          lcd_printPGM(PSTR("SD"));
+          lcd.print(itostr3(card.percentDone()));
+          lcd.write('%');
+        }
+        else {
+          lcd_printPGM(PSTR("Ex"));
```

Comment:
Fixed English abbreviations are unfriendly for internationalization purposes. If this is supposed to stand for "Extrusion" I'm not sure it's the most well-understood substitution (English or otherwise) for "Flow." Personally, I do not want this information on my screen. So (as with all new features) this should be made an Optional Behavior by adding a new configuration option to `Configuration_adv.h`.